crates/storage/provider/src/providers/database/provider.rs

```diff
@@ -575,7 +607,7 @@ impl<TX: DbTxMut + DbTx> DatabaseProvider<TX> {
         let block_bodies = self.get_or_take::<tables::BlockBodyIndices, false>(range)?;
 
         if block_bodies.is_empty() {
-            return Ok(Vec::new())
+            return Ok(Vec::new());
```

Comment:
please revert these lint changes, and run clippy with nightly toolchain. try also `make lint`.

Reply:
fixing that today!

Reply:
I've removed all the unnecessary formatting